exasol_data_science_utils_python/schema/column_name.py

```diff
@@ -30,3 +30,6 @@ def __eq__(self, other):
 
     def __repr__(self):
         return generate_repr_for_object(self)
+
+    def __hash__(self):
+        return hash((self.name, self._table_name))
```

Comment:
I have bad experience with having manual hashing functions, when you add members you easily forget to change the hash function and then error searching can cosume much time.
```suggestion
        return hash(self.__dict__.values(self.name, self._table_name))
```